stephan-livera-podcast/2018-07-28-Vijay Boyapati-Stephan Livera.md

```diff
@@ -0,0 +1,216 @@
+---
+title: How Bitcoin ‘mentally captures’ people.
+transcript_by: Dereje Hinsermu
+categories: ['podcast']
+speaker: ['Vijay Boyapati']
+date: 2018-07-28
+media: https://stephanlivera.com/episode/2/
```

Comment:
```suggestion
media: https://stephanlivera.com/download-episode/507/2.mp3
```

This should be a link to the media file rather than the landing page.